pom.xml

```diff
@@ -67,7 +67,7 @@
         <maven.compiler.target>11</maven.compiler.target>
         <asm.version>9.3</asm.version>
         <httpclient.version>4.5.13</httpclient.version>
-        <jetty.version>9.4.48.v20220622</jetty.version>
+        <jetty.version>9.4.49.v20220914</jetty.version>
         <log4j.version>2.18.0</log4j.version>
```

Comment:
HtmlUnit declares a dependency to log4j 2.19.0. Should we bump it too?

Reply:
Done.